roles/ceph-common/tasks/installs/install_on_redhat.yml

```diff
@@ -39,13 +39,66 @@
   include: redhat_ceph_repository.yml
   when: ceph_origin == 'upstream'
 
+- name: make temp dir
+  file: path=/tmp state=directory
```

Comment:
The name of the task is a bit confusing, are we trying to create a 'real' tempory directory (ie: `mktemp -d`) or just `/tmp`?

The task should be written like this: 

```
file:
  path: /tmp
  state: directoty
```